pkg/local-storage/member/node/storage/executor_lvm.go

```diff
@@ -756,7 +757,7 @@ func (lvm *lvmExecutor) lvcreate(lvName string, vgName string, options []string)
 		CmdArgs: append(options, vgName, "-n", lvName, "-y"),
 	}
 	res := lvm.cmdExec.RunCommand(params)
-	if res.ExitCode == 0 {
+	if res.ExitCode == 0 || strings.Contains(res.ErrBuf.String(), ErrorLocalVolumeExistsInVolumeGroup.Error()) {
```

Comment:
lgtm

This error check also exist at [here](https://github.com/hwameistor/hwameistor/blob/445f617458f8b2102196d165e40a7f73f6f6c461/pkg/local-storage/member/node/storage/executor_lvm.go#L283), for simplicity, can you remove the code from there

Reply:
done

Reply:
> lgtm
> 
> This error check also exist at [here](https://github.com/hwameistor/hwameistor/blob/445f617458f8b2102196d165e40a7f73f6f6c461/pkg/local-storage/member/node/storage/executor_lvm.go#L283), for simplicity, can you remove the code from there

done